sklearn/datasets/mldata.py

```diff
@@ -24,7 +24,7 @@
 
 
 @deprecated('mldata_filename was deprecated in version 0.20 and will be '
-            'removed in version 0.22')
+            'removed in version 0.22 . Please use fetch_openml. ')
```

Comment:
please get rid of the redundant spaces, i.e., before the first ``.`` and after the second ``.``